README.adoc

```diff
@@ -1,4 +1,4 @@
-//  Copyright (c) 2017, 2021 IBM Corporation and others.
+//  Copyright (c) 2017, 2022 IBM Corporation and others.
```

Comment:
general comment: although JAX-RS was an abbreviation that was associated with both the Liberty feature and the java spec, restfulWS only seems to apply to the Liberty feature. The [Jakarta RESTful Web Services spec](https://jakarta.ee/specifications/restful-ws/3.0/jakarta-restful-ws-spec-3.0.html) doesn't use the restfulWS  abbreviation at all. So it seems strange to refer to "Creating a restfulWS application". I don't think we'd want to use that abbreviation unless we were referring specifically to enabling/configuring the Liberty restfulWS feature. So you might want to say "Creating a RESTful Web Services application" and similar throughout. Some from dev might be able to clarify further





[1:36](https://ibm-cloud.slack.com/archives/DP92HKKM3/p1644259009495549)
Here's the contextual statement we added to the docs for JAX-RS and restfulWS features:
Jakarta RESTful Web Services, formerly known as JAX-RS, is a [Jakarta EE](https://www.openliberty.io/docs/22.0.0.1/jakarta-ee.html) platform API. In Open Liberty 21.0.0.12 and later, you can use Jakarta [RESTful Web Services 3.0](https://www.openliberty.io/docs/latest/reference/feature/restfulWS.html) functions by enabling the RESTful Web Services 3.0 feature. The JAX-RS 2.0 and 2.1 implementations are available through the Java RESTful Services feature ([jaxrs-2.0](https://www.openliberty.io/docs/latest/reference/feature/jaxrs-2.0.html) and [jaxrs-2.1](https://www.openliberty.io/docs/latest/reference/feature/jaxrs-2.1.html)). The change in feature name and version reflects the change in API package name prefixes from javax.* to jakarta.*, which is common to all Jakarta EE 9.1 features. For more information, see [Jakarta RESTful Web Services 3.0 behavior changes](https://www.openliberty.io/docs/22.0.0.1/reference/jaxrs-dif.html).